### PR TITLE
add DUID-UUID

### PR DESCRIPTION
**Pull request by pmazzini**

### CODE REVIEW

File: dhcpv6/duid.go [hunk 1]
```diff
@@ -10,15 +10,17 @@ import (
 type DuidType uint16
 
 const (
-	DUID_LLT DuidType = 1
-	DUID_EN  DuidType = 2
-	DUID_LL  DuidType = 3
+	DUID_LLT  DuidType = 1
```
[the remark this answers is not in the document]
Comment (insomniacslk) [reply dhcpv6/duid.go]: I will address all these at some point, but for now let's just ignore it

File: dhcpv6/duid.go [hunk 2]
```diff
@@ -28,6 +31,7 @@ type Duid struct {
 	LinkLayerAddr        []byte
 	EnterpriseNumber     uint32 // for DUID-EN. Ignored otherwise
 	EnterpriseIdentifier []byte // for DUID-EN. Ignored otherwise
+	Uuid                 [16]byte // for DUID-UUID. Ignored otherwise
```
Comment (insomniacslk) [inline dhcpv6/duid.go]: I'd rather have a `[]byte` and do the appropriate checks elsewhere in the code. My experience with arrays in this type of use is that it's more painful to cast/convert things around than just dealing with making sure that we have the right number of bytes

Comment (pmazzini) [reply dhcpv6/duid.go]: Ok, I'll update it.

File: dhcpv6/duid.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (pmazzini) [reply dhcpv6/duid.go]: I think it is no longer complaining after adding a comment on the block.

**Pull request merged**